AmazonChimeSDK/AmazonChimeSDK/audiovideo/DefaultAudioVideoController.swift

```diff
@@ -94,4 +103,14 @@ import Foundation
     public func stopRemoteVideo() {
         videoClientController.stopRemoteVideo()
     }
+
+    @objc private func handleMediaServiceReset(notification: Notification) {
+        DispatchQueue.main.asyncAfter(deadline: .now() + .seconds(1)) {
```

Comment:
why the 1sec delay?

Reply:
Discussed with Nick, wanted to release audio when reset is completed. Tested with no delay, it works the same though.

Reply:
I tested a few more times without delay, no unexpected behaviors. Will remove it.